lib/mayaUsd/render/pxrUsdMayaGL/sceneDelegate.cpp

```diff
@@ -80,6 +80,14 @@ class PxrMayaHdShadowMatrix : public HdxShadowMatrixComputation
         _shadowMatrices = light.GetShadowMatrices();
     }
 
+#if HDX_API_VERSION >= 8
+    std::vector<GfMatrix4d>
+    Compute(const CameraUtilFraming& framing, CameraUtilConformWindowPolicy policy) override
+    {
+        return _shadowMatrices;
+    }
```

Comment:
Both overloads of `Compute()` will exist for a while during a transition period until the old one that takes a `GfVec4f` is eventually removed.